### ISSUE TITLE
Install sct_testing_data before running pytest

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#2775]
This testing dataset is required by some tests under `unit_tests/`. This dataset is installed by Travis during CI, but it would be nice to have it installed when launching pytest on local stations. This is anticipating dropping `sct_testing` at some point (could be replaced by pytest).

**Comment by cfhammill** [comment]
should this happen by default during every installation? Or we could make a special `conftest.py` that downloads the data if it isn't there already before starting the tests to spare the download until someone wants to run the tests. https://stackoverflow.com/questions/19225279/pytest-init-setup-for-few-modules

**Comment by jcohenadad** [comment]
> Or we could make a special conftest.py that downloads the data if it isn't there already before starting the tests to spare the download until someone wants to run the tests. 

yup! i like that. 

however i would make the download systematic (even if already exists). why? in case the folder is there, but the data have changed
